sirepo/package_data/static/js/controls.js

```diff
@@ -745,34 +748,35 @@ SIREPO.app.directive('latticeFooter', function(appState, controlsService, lattic
                         })
                         .on('dblclick', () => elementClicked(values[0]))
                         .appendTo($('.sr-lattice-holder'));
-                    let divPos = {
-                        rect: div[0].getBoundingClientRect(),
-                        // need to track offset manually
```

Comment:
@moellep I didn't find this to be true. I tried in Chrome and FF. Will you test and see what happens?